jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package jme3test.asset;
+
+import com.jme3.app.SimpleApplication;
+import com.jme3.light.DirectionalLight;
+import com.jme3.math.Vector3f;
+import com.jme3.scene.Spatial;
+
+/**
+ * Tests issue 2083.
+ * 
+ * Vertex colors on gltf models was not being enabled.
```

Comment:
Test documentation should indicate how to determine whether the test passed or failed.